dojo/tools/blackduck/importer.py

```diff
@@ -54,12 +55,13 @@ def _process_zipfile(self, report):
         security_issues = dict()
         try:
             with zipfile.ZipFile(str(report)) as zip:
-                for file_name in zip.namelist():
-                    if file_name.endswith('files.csv'):
-                        with io.TextIOWrapper(zip.open(file_name)) as f:
+                for full_file_name in zip.namelist():
+                    file_name = full_file_name.split("/")[-1]
+                    if 'source' in file_name:
```

Comment:
Wasn't it `sources` you had mentioned in Slack and not `source`? Also of course, please think there about backward-compatibility.

Reply:
Turns out the file is called "source".
Example: source_2019-11-25-223934.csv

Also copy that, I will consider backwards compatibility for these.